aten/src/ATen/native/ConvUtils.h

```diff
@@ -62,10 +63,14 @@ enum class ConvBackend {
 
 // Function to select the convolution backend based on the inputs and params.
 // This overload is used within the convolution internals but not exposed to python.
+// NB: The forward pass provides a bias tensor while the backward pass provides
+// a bool indicating whether the bias is defined. This is done to save memory by
+// avoiding saving the full bias tensor for backward.
 TORCH_API ConvBackend select_conv_backend(
     const Tensor& input,
     const Tensor& weight,
-    const Tensor& bias,
+    const c10::optional<IntArrayRef> bias_sizes_opt,
+    const bool need_backward,
```

Comment:
This would need a comment explaining what it means is?
I guess it is: grad mode is enabled and either input, weight or bias requires grad. ?

Also what about when this is called during backward? This means double backward will be needed?